addons/website/views/res_config_views.xml

```diff
@@ -7,134 +7,384 @@
             <field name="model">website.config.settings</field>
             <field name="arch" type="xml">
                 <form string="Configure Website" class="oe_form_configuration">
-                    <header>
-                        <button string="Apply" type="object" name="execute" class="oe_highlight"/>
+                    <header class="o_header_sticky">
```

Comment:
This class has been defined by india teams, the bar with buttons become sticky (see sale settings task).